Model/Customer/Customer.php

```diff
@@ -91,7 +91,7 @@ public function getUpdatedAt()
     /**
      * @inheritDoc
      */
-    public function setCustomerId(int $customerId)
+    public function setCustomerId(?int $customerId)
```

Comment:
Can be null now, since the customer can be not_logged_in and only afterwards log in/create account